app/controllers/api/levels_controller.rb

```
@@ -0,0 +1,32 @@
+
+class API::LevelsController < ApplicationController
+  before_filter :ensure_staff?
+
+
+  # PUT api/level/:id
+  def update
+    level = Level.find(params[:id])
+    level.criterion.remove_expectations if updating_meets_expectations?
+
+    if level.update_attributes(level_params)
+      render json: { message: "level successfully updated", success: true }
+    else
+      render json: {
+        errors: [{ detail: "failed to update level" }], success: false
```

Do we want to return the actual errors from the model here so the user can know what happened?